js/test/components/operatorSignIn/operatorSignInModal.test.tsx

```diff
@@ -122,6 +130,54 @@ describe("OperatorSignInModal", () => {
     await userEvent.click(
       view.getByRole("button", { name: "Complete Fit for Duty Check" }),
     );
+    expect(fetchMock).toHaveBeenCalledWith(
+      "/api/signin",
+      expect.objectContaining({
+        body: expect.stringMatching(/"override":null/),
+      }),
+    );
+    expect(view.getByText("signed in successfully")).toBeInTheDocument();
+  });
+
+  test("submits successful attestation to the server with override", async () => {
+    putMetaData("csrf-token", "TEST-CSRF-TOKEN");
+    const fetchMock = jest.mocked(fetch).mockResolvedValueOnce({
+      ok: true,
+    } as Response);
+
+    const view = render(
+      <OperatorSignInModal
+        show={true}
+        onComplete={jest.fn()}
+        close={jest.fn()}
+      />,
+    );
+    await userEvent.type(view.getByRole("textbox"), "1234");
```

Comment:
naming nitpick:

Reading this test it took me a long time to figure out how it mocks the expired cert. Now I see it's based on the badge number, "123" is certified and "1234" has an expired cert.

I think it'd be easier to follow the test if the operator had a badge string like `"expiredcert"` or something. Or, alternatively, a comment on this line explaining that this badge number is controlling which cert to get, so that anybody editing this test in the future can tell this string is important.

Reply:
added a comment above it 👍 https://github.com/mbta/orbit/pull/220/files#diff-cb55066e3d310eb5b472f93dd2e0ff0f71aa20d9b3b346c5a7bb8b40f25b68d5R155